src/PhpHooks/Command/ForbiddenCommand.php

```diff
@@ -53,4 +53,25 @@ public function run(InputInterface $input, OutputInterface $output)
             }
         }
     }
+
+    /**
+     * @param string $filePath
+     * @param array $configuration
+     * @return bool
+     */
+    protected function checkExcludeFile($filePath, array $configuration)
+    {
+        if (empty($configuration['exclude']) || !is_array($configuration['exclude'])) {
```

Comment:
$configuration['exclude'] should always be an array. You can force this in the configuration class.